packages/client/src/components/Location/LocationSection.tsx

```diff
@@ -64,23 +64,23 @@ const LocationSection: FunctionComponent<SectionComponent> = (props) => {
     // eslint-disable-next-line
   }, [checker, isActive, skipLocationSection]);
 
-  useEffect(() => {
-    // Autocomplete the location section in case the NewCheckerModal has been used with a stored address
-
-    // Detect if the user has used the NewCheckerModal with a stored address
-    if (address && !sectionData.length && !autoCompleteSection) {
-      setAutoCompleteSection(true);
-    }
-
-    // Wait until the sectionData has been set by CheckerPage and go to the next section
-    if (isActive && sectionData.length && autoCompleteSection) {
-      goToNextSection();
-      setAutoCompleteSection(false);
-    }
-    // eslint-disable-next-line
-  }, [isActive, sectionData.length, autoCompleteSection]);
-
-  if (!checker || !sectionData || autoCompleteSection || skipLocationSection) {
+  // useEffect(() => {
```

Comment:
The part above (see `if (address) {`) is why the checker firesafety is broken when restarted. That should be a quick fix